app/assets/src/components/ui/controls/dropdowns/MultipleNestedDropdown.jsx

```diff
@@ -214,32 +218,48 @@ class MultipleNestedDropdown extends React.Component {
 
   renderLabel() {
     const numberOfSelectedOptions = this.getNumberOfSelectedOptions();
+
+    const label = numberOfSelectedOptions > 0 && (
+      <DropdownLabel
+        className={cs.dropdownLabel}
+        disabled={this.props.disabled}
+        text={String(numberOfSelectedOptions)}
+      />
+    );
+
     return (
-      <div className="label-container">
-        <div className="label-container-title">{this.props.label}</div>
-        {numberOfSelectedOptions > 0 && (
-          <Label className="label-container-count">
-            {numberOfSelectedOptions}
-          </Label>
-        )}
-      </div>
+      <DropdownTrigger
+        className={cs.dropdownTrigger}
+        label={this.props.label}
+        value={label}
+        rounded={this.props.rounded}
+      />
     );
   }
 
   render() {
-    const { selectedOptions, selectedSuboptions, ...props } = this.props;
+    const otherProps = omit(
```

Comment:
We should add this pattern of using props and removing them to the dev guidelines.